client/components/standard-deposit-button/style.scss

```diff
@@ -0,0 +1,3 @@
+.wcpay-standard-deposit-modal__footer > * + * {
```

Comment:
This is to add a small margin between the modal footer buttons. I'm not sure if this is the intended use of these variables but I've seen them used in various places and it works in this case.

https://github.com/Automattic/woocommerce-payments/blob/3b9c83f4892d272b5f1051743eb97472dffeb131/client/stylesheets/abstracts/_variables.scss#L5